### ISSUE TITLE
Change animate mode default

### ISSUE THREAD

**Issue by rreusser** [plotly/plotly.js#1076]
I _usually_ change the default animation mode to `immediate`. As a result of user input, this seems like the more natural and expected behavior. `afterall` results in a lot of queuing and seems like a more obscure use-case. I think `immediate` would be a better default. Is this changable?

**Comment by etpinard** [comment]
One could make a point that animations aren't very much in the wild yet - so a _breaking_ default change wouldn't be the end of the world.

But, I'd vote for waiting for v2.

**Comment by rreusser** [comment]
Fore reference, I ran into a weird result with bindings where a bunch of animation commands coming through *without* `mode: 'immediate'` results in them getting queued and dispatched later on. The result is that the slider position changes to match the current frame after you've stopped dragging. This is a little tricky to prevent since we're not propagating the triggering event all the way through the animate command so that it's difficult to figure out whether the slider is changing itself or whether a different command has changed its value.

Long story short, the default behavior of the slider is a bit weird and unpleasant with components and fixing it is much more nuanced and challenging than a better default.

**Comment by rreusser** [comment]
(But it's a simple change so is not a big deal)

**Comment by gvwilson** [comment]
Hi - this issue has been sitting for a while, so as part of our effort to tidy up our public repositories I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our stack. Cheers - @gvwilson
